sentry-android-integration-tests/sentry-uitest-android/src/androidTest/java/io/sentry/uitest/android/EnvelopeTests.kt

```diff
@@ -115,13 +117,19 @@ class EnvelopeTests : BaseUiTest() {
                 assertContains(transactions.map { t -> t.id }, transactionItem.eventId.toString())
                 val startTimes = transactions.map { t -> t.relativeStartNs }
                 val endTimes = transactions.mapNotNull { t -> t.relativeEndNs }
+                val startCpuTimes = transactions.map { t -> t.relativeStartCpuMs }
+                val endCpuTimes = transactions.mapNotNull { t -> t.relativeEndCpuMs }
                 assertNotEquals(startTimes[0], startTimes[1])
                 assertNotEquals(startTimes[0], startTimes[2])
                 assertNotEquals(startTimes[1], startTimes[2])
                 assertNotEquals(endTimes[0], endTimes[1])
                 assertNotEquals(endTimes[0], endTimes[2])
                 assertNotEquals(endTimes[1], endTimes[2])
 
+                // The cpu timestamps shouldn't be all the same
+                assertFalse(startCpuTimes[0] == startCpuTimes[1] && startCpuTimes[1] == startCpuTimes[2])
+                assertFalse(endCpuTimes[0] == endCpuTimes[1] && endCpuTimes[1] == endCpuTimes[2])
```

Comment:
`m`: I think we could do a more extensive check here. We could verify that one is bigger than the other, and we could also verify that they are in a certain time range.

Reply:
some timestamps could be the same, so we should check if one is <= than the other.
But then, the transaction list is not ordered, due to how they are stored (we use a map instead of a list to be able to quickly check the existence of a certain id)
So i should order the list and then check the each timestamp is lower or equals to the following one.
I'll add a comment on it to explain it